emoncms/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN \
         php7-session=7.4.22-r0 \
         php7-zip=7.4.22-r0 \
         php7=7.4.22-r0 \
+        redis=6.2.5-r0 \
```

Comment:
Latest version available is now `6.2.6-r0`. See [repository](https://pkgs.alpinelinux.org/packages?name=redis&branch=edge).